website/src/documentation/sdks/python-dependencies.md

```diff
@@ -26,460 +26,8 @@ behavior in the service. If you are using any of these packages in your code, be
 aware that some libraries are not forward-compatible and you may need to pin to
 the listed versions that will be in scope during execution.
 
-<p>To see the compile and runtime dependencies for your Beam SDK version, expand
-the relevant section below.</p>
+To see the compile and runtime dependencies for your Beam SDK version, core SDK dependencies are listed in `https://raw.githubusercontent.com/apache/beam/v<version_number>/sdks/python/setup.py` under `REQUIRED_PACKAGES`.
 
-<details><summary markdown="span"><b>2.17.0</b></summary>
+If you installed any [extra features]({{ site.baseurl }}/get-started/quickstart-py#extra-requirements) along with `apache-beam`, such as `gcp` or `test`, you should review the lists under `REQUIRED_TEST_PACKAGES`, `GCP_REQUIREMENTS`, or `INTERACTIVE_BEAM` for additional dependencies. 
```

Comment:
s/"you can"/w/"you should" since you don't really have to review the lists even if you installed the extra features

Also, do we have links to the REQUIRED_TEST_PACKAGES/GCP_REQUIREMENTS/INTERACTIVE_BEAM lists?

Reply:
Made the sed change. 

The other lists live within the setup.py file right below `REQUIRED_PACKAGES`.

Reply:
I would say: "If you require extra features ... such as ...,you should review ... for additional dependencies"